### PR TITLE
Added code and documentation

### PR DESCRIPTION
**Pull request by amalshaji**
Added code and documentation of problem number 6 on Project Euler. This code is tested positive for both test cases on Project Euler.

### CODE REVIEW

File: 002/002.go [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-			a = b
-			b = c
-		}
-
-		fmt.Println(sum)
-	}
-}
+    var t int
+    fmt.Scanf("%d", &t)
+    if t >= 1 && t <= 100000 {
```
Comment (amalshaji) [inline 002/002.go:10]: Please make sure that you add the constraints for the variable t and n given in the problem statement.

**Review by ameykhale — COMMENTED** [review]
002.go still fails that one test case. What's up with the `$`'s and `&nbsp` in 006's README.md?

**Comment by amalshaji** [comment]
$'s and &nbsp to make the markdown look better for the documentation.

**Comment by amalshaji** [comment]
> 002.go still fails that one test case. What's up with the `$`'s and `&nbsp` in 006's README.md?

**Pull request closed without merging**